code/modules/mob/living/carbon/human/species.dm

```diff
@@ -306,6 +310,15 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		for(var/i in inherent_factions)
 			C.faction += i //Using +=/-= for this in case you also gain the faction from a different source.
 
+	if(flying_species && isnull(fly))
+		fly = new
+		fly.Grant(C)
+	if(C.dna && C.dna.species && (C.dna.features["wings"] != wings_icon))
```

Comment:
This doesn't make sense. C.dna.species is src.

Reply:
honestly, if it's a species initial thing, then the wings should just be defined and not set, ya